app/services/cosmos/cosmos.utils.ts

```diff
@@ -50,3 +69,133 @@ export function calculateUnbondingDelegationBalanceFromResultEntries(entries: Co
 export function validateAccountAddress(address: string) {
   return /^cosmos1[ac-hj-np-z02-9]{38}$/.test(address)
 }
+
+export function convertDecCoin(coin: DecCoin): CosmosCoinResult {
+  return coin as CosmosCoinResult
```

Comment:
For simple conversion like this, we can just cast it inline instead of writing a function?

Reply:
I see, will remove this function.